app/src/main/java/com/haishinkit/studio/PlaybackTabFragment.kt

```diff
@@ -71,6 +71,9 @@ class PlaybackTabFragment: Fragment(), IEventListener {
             stream.play(Preference.shared.streamName)
         }
     }
+    
+    fun unusedMehtod() {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/shogo4405/HaishinKit.kt/blob/1a4d0f274065ad6c0d0770ad005432183d879eb6/app/src/main/java/com/haishinkit/studio/PlaybackTabFragment.kt#L75"></span>Function "unusedMehtod" is never used</td>
    </tr>
  </tbody>
</table>